### PR TITLE
[CEL] Update the apiVersion

### PR DESCRIPTION
**Pull request by jerop**
# Changes

Update the apiVersion from `cel.dev/v1alpha1` to `cel.tekton.dev/v1alpha1`

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Commit messages includes a project tag in the subject - e.g. [OCI], [hub], [results], [task-loops]

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md) for more details._

/cc @pritidesai

### CODE REVIEW

**Review by pritidesai — APPROVED** [review]
thanks @jerop for fixing this 👍

File: cel/README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ cel-controller-654bdc4cc8-7bvvn   1/1     Running   0          3m4s
 ## Usage
 
 To evaluate a CEL expressions using `Custom Tasks`, we need to define a [`Run`](https://github.com/tektoncd/pipeline/blob/master/docs/runs.md)
-type with `apiVersion: cel.tekton.dev/v1alpha1` and `kind: CEL`. The `Run` takes the CEL expressions to be evaluated
+type with `apiVersion: cel.dev/v1alpha1` and `kind: CEL`. The `Run` takes the CEL expressions to be evaluated
```
Comment (ImJasonH) [inline cel/README.md:35]: I'm not sure we should squat domains we don't control. `cel.tekton.dev` sounded fine to me, or I've been using `example.dev` while it's experimental.

Comment (jerop) [reply cel/README.md]: Makes sense, fixed it to be `cel.tekton.dev` everywhere

**Review by ImJasonH — APPROVED** [review]

**Pull request merged**